pkg/kotsclient/get_release.go

```diff
@@ -0,0 +1,50 @@
+package kotsclient
+
+import (
+	releases "github.com/replicatedhq/replicated/gen/go/v1"
+	"github.com/replicatedhq/replicated/pkg/graphql"
+)
+
+type GraphQLResponseKotsGetRelease struct {
+	Data   KOTSReleaseResponseData `json:"data,omitempty"`
+	Errors []graphql.GQLError      `json:"errors,omitempty"`
+}
+
+type KOTSReleaseResponseData struct {
+	KotsReleaseData *KOTSReleaseWithSpec `json:"kotsReleaseForSequence"`
+}
+
+type KOTSReleaseWithSpec struct {
+	Sequence     int64  `json:"sequence"`
+	ReleaseNotes string `json:"releaseNotes"`
+	Spec         string `json:"spec"`
+}
+
+const kotsQueryGetReleaseForSequence = `
+query kotsReleaseForSequence($appId: ID!, $sequence: Int!) {
```

Comment:
is this really only available as graphql? If so, ok

Reply:
yeah trust me I had the same question 😂